samples/event-logger/src/components/InputBoard/Filter/Filter.js

```diff
@@ -0,0 +1,137 @@
+import {Component} from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import Heading from '@enact/sandstone/Heading';
+import Picker from '@enact/sandstone/Picker';
+import SwitchItem from '@enact/sandstone/SwitchItem';
```

Comment:
Please re-order these based on our coding rule.

Reply:
It's fixed.